src/tools/perf/perftest.c

```diff
@@ -324,11 +324,12 @@ static void usage(struct perftest_context *ctx, const char *program)
     printf("     -x <tl>        Transport to use for testing.\n");
     printf("     -c <cpu>       Set affinity to this CPU. (off)\n");
     printf("     -n <iters>     Number of iterations to run. (%ld)\n", ctx->params.max_iter);
-    printf("     -s <size>      Message size. (%zu)\n", ctx->params.message_size);
+    printf("     -s <size>      Message size. default is (%zu). It can contains (applicable in zcopy data layout only) a list of message sizes, separated by comma. For example, \"-s 16,48,8192,8192,14\"\n", ctx->params.message_size);
     printf("     -H <size>      AM Header size. (%zu)\n", ctx->params.am_hdr_size);
     printf("     -w <iters>     Number of warm-up iterations. (%zu)\n", ctx->params.warmup_iter);
     printf("     -W <count>     Flow control window size, for active messages. (%u)\n", ctx->params.uct.fc_window);
     printf("     -O <count>     Maximal number of uncompleted outstanding sends. (%u)\n", ctx->params.max_outstanding);
+    printf("     -i <count>     Stride between beginnings of payload elements in the buffer in bytes (applicable in zcopy data layout only). The same as UCT uct_iov_t stride.\n");
```

Comment:
This is not for "zcopy" only, it would be used for UCP as well.
"Stride between consecutive buffers in the same message. 0 means the buffers are allocated contiguously. Otherwise, must not be smaller than any of the buffer sizes specified by -s"